Generic/main.cpp

```diff
@@ -70,14 +70,14 @@ extern "C" {
 
 
 /* Check if the compiler thinks you are targeting the wrong operating system. */
-#if defined(__linux__)
-#error "You are not using a cross-compiler, you will most certainly run into trouble"
-#endif
+// #if defined(__linux__)
```

Comment:
I'm not sure what to replace this with. Please advise if you know.  This is an artifact of using the built in compiler.

Reply:
I mean I think this stuff is just here to stop noobs doing the wrong thing. You don't need to defend against this.